### PR TITLE
Update Filebeat module dev guide

### PR DESCRIPTION
**Pull request by webmat**
- Replace export_dashboard example to use the recommended `-yml` option
- Add a link to export_dashboard's doc

### CODE REVIEW

**Comment by kvch** [comment]
Could you please rebase this PR? The header problems should be fixed on master.

File: docs/devguide/modules-dev-guide.asciidoc [hunk 1]
```diff
@@ -74,16 +74,35 @@ is used as a title in the docs, so it's best to capitalize it.
...
+
+Then run `export_dashboards` like this:
 
 [source,shell]
 ----
 $ cd dev-tools/cmd/dashboards
-$ make # if export_dashboard is not built
-$ ./export_dashboards -dashboard '{dashboard-id}' -output '../../../filebeat/module/{module}/_meta/kibana/default/dashboard'
+$ make # if export_dashboard is not built yet
+$ ./export_dashboards -yml '../../../filebeat/module/{module}/module.yml'
```
Comment (ruflin) [inline docs/devguide/modules-dev-guide.asciidoc]: I wonder if we should instead recommend to use `go run dev-tools/cmd/dashboards/export_dashboards ...` so make is not needed and it always uses the local platform.

Comment (kvch) [reply docs/devguide/modules-dev-guide.asciidoc]: At this point it is acceptable here, as `make` is not yet eliminated from module generation. Also, I think calling `./export_dashboards ...` is more elegant in the documentation.

**Comment by webmat** [comment]
Rebase done and pushed. So we're good with keeping the `make` step and using the compiled `export_dashboards`? I also think it's nicer to have this much less to type ;-)

File: docs/devguide/modules-dev-guide.asciidoc (hunk 1 above)
Comment (ruflin) [reply docs/devguide/modules-dev-guide.asciidoc]: Both works for me. The command is shorter but you need to get all the `../` right :-)

**Comment by ruflin** [comment]
Both options work for me as long as it's working :-)

**Comment by webmat** [comment]
Ok thanks. Merge at will when ready :-)

**Pull request merged**